unique_toolkit/unique_toolkit/language_model/service.py

```diff
@@ -255,6 +328,16 @@ async def stream_complete_async(
         Returns:
             The LanguageModelStreamResponse object once the stream has finished.
         """
+        validate_required_parameters(
```

Comment:
Is this call required here because each method of this class has a different set of required parameters?

Reply:
yes, it accepts an arbitrary set of parameters which is why I can reuse across functions and classes